build.yaml

```diff
@@ -146,6 +145,6 @@ TaskHeader: |
 GitHubConfig:
   GitHubFilesToAdd:
     - 'CHANGELOG.md'
-  GitHubConfigUserName: dscbot
-  GitHubConfigUserEmail: dsccommunity@outlook.com
+  #GitHubConfigUserName: dscbot
+  #GitHubConfigUserEmail: dsccommunity@outlook.com
```

Comment:
@gaelcolas hat do you want me to use for these two? Same as Sampler, or do you want to set up another account for this (sending in the change log PR)?